src/test/java/seedu/address/testutil/TypicalProperties.java

```diff
@@ -27,40 +27,40 @@
  */
 public class TypicalProperties {
 
-    public static final Property ALICE = new PropertyBuilder().withName("Jurong West Ave 6")
+    public static final Property P_ALICE = new PropertyBuilder().withName("Jurong West Ave 6")
             .withSeller("Alice Pauline")
             .withPrice("654321")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253")
             .withTags("HDB").build();
-    public static final Property BENSON = new PropertyBuilder().withName("Mayflower")
+    public static final Property P_BENSON = new PropertyBuilder().withName("Mayflower")
             .withSeller("Benson Meier")
             .withPrice("654321")
             .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
             .withTags("condo").build();
-    public static final Property CARL = new PropertyBuilder().withName("Carl Kurz").withPhone("95352563")
+    public static final Property P_CARL = new PropertyBuilder().withName("Carl Kurz").withPhone("95352563")
```

Comment:
Currently, P_CARL, P_DANIEL, P_ELLE etc. do not have a withSeller, should we add those? Additionally, their names are still human names, not property names. This may not be a priority if we are only mainly testing with alice and benson.
This can be left as a TODO.